libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -104,6 +104,56 @@ string YulUtilFunctions::copyToMemoryFunction(bool _fromCalldata)
 	});
 }
 
+string YulUtilFunctions::requireOrAssertFunction(bool _assert, Type const* _messageType)
+{
+	string functionName =
+		string(_assert ? "do_assert" : "require") +
```

Comment:
I think you have to "escape" `require` instead of `assert` - but probably escaping both would be a good idea. What about `x_helper`?